assets/css/abstracts/_mixins.scss

```diff
@@ -1,10 +1,14 @@
-// Converts a font-size in px to rem and optionally sets a relative line-height.
-@mixin font-size($sizeValue: 16px, $lineHeight: false) {
-	font-size: ($sizeValue / 16px) * 1rem;
-
-	@if ($lineHeight) {
-		line-height: $lineHeight / $sizeValue;
-	}
+$fontSizes: (
+	"smaller": 0.75,
+	"small": 0.875,
+	"regular": 1,
+	"large": 1.25,
+	"larger": 2,
+);
+
+// Maps a named font-size to its em equivalent.
+@mixin font-size($sizeName) {
+	font-size: map-get($fontSizes, $sizeName) * 1em;
```

Comment:
Just a small question on mixin usage. Are there any advantages to the mixin over variables?

i.e. 
```
@include font-size(larger);
```
vs
```
font-size: $larger;
```

Reply:
A mixin gives us more control, so the idea is that it will allow us to easily migrate all font sizes to support global styles in the future. There is an example here: https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/2403#issuecomment-624056776.

That example is a bit old (it uses px→rem instead of named→em) and global styles are still in development so its shape might be different from the one in that example, but in general I think having all font sizes in a mixin will eventually make it easier to embrace global styles.

However, I acknowledge that with search & replace we can easily switch between `@include font-size(larger);` ←→ `font-size: $larger;`, so I don't have a strong preference on any of them other than in the future we might be forced to use mixins.

Reply:
Can we support both? Readability I prefer standard font-size. In no means a blocker though :)

Reply:
Hmm, I'm a bit on the edge with this. While I don't have any strong preference between both systems, I would really prefer that we stick with one. It really simplifies things when searching for code.

I will merge this as-is for now, and we can migrate in a follow.